ocs_ci/deployment/deployment.py

```diff
@@ -1335,6 +1335,14 @@ def deploy_ocs_via_operator(self, image=None):
                 cluster_data["spec"]["encryption"] = {
                     "storageClassName": storageclassnames["encryption"]
                 }
+        # Bluestore for RDR greenfield deployments: 4.14 onwards
+        if config.multicluster and (
```

Comment:
If it's relevant only from 4.14, because of upgrade scenario we are running deployment of previous version of ODF from OCS-CI branch of version we are upgrading to, we would like to add condition here to happen only if OCS_VERSION is >= 4.14. Otherwise it will do this configuration in 4.13 deployment.

Reply:
done